### ISSUE TITLE
Unable to switch profile

### ISSUE THREAD

**Issue by Yoiink** [Noitidart/Profilist#47]
I'm unable to switch profile on 55.0b2 (32-bit) (Developer Edition Firefox). 

The only way I believe you can do this is by hovering over the current profile on the menu which currently isn't showing the menu of profiles as it used to.

**Comment by Noitidart** [comment]
Hi Yoiink, I got a report that Profilist is no longer working in Firefox 55. I will have to dive it to figure out if it's worthwhile to fix it, or just wait on the webext remake.

**Comment by Noitidart** [comment]
whoops accidentally closed. reopened

**Comment by Yoiink** [comment]
No problem, glad you're aware of the issue. I hope you can fix it since it's super useful to me!

**Comment by Noitidart** [comment]
Thanks so much for your comment! I really love to hear people are using this! :)

**Comment by geekyi** [comment]
**tldr** would like to help get this working asap, what can I do? But I'm not an expert even if I did make a hello world xul extension a long time ago..

This has literally become the no.1 addon I install (also happens as a side effect of how it works, but that's another matter 😋), because this allows creation of new profiles to test when something is wrong with 1-click.

Now that it's not working along with I don't know how many other addons; firefox has literally(!) become useless for me. I use a binary search to eliminate problematic addons, and this is the first thing I need working

**Comment by geekyi** [comment]
Is there a gitter.im chat or even irc?

**Comment by xeniun** [comment]
Up.
Me too.

**Comment by TerkiKerel** [comment]
Up
